src/System.Threading.Overlapped/src/System.Threading.Overlapped.builds

```diff
@@ -2,9 +2,16 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="System.Threading.Overlapped.CoreCLR.csproj">
```

Comment:
This got merged so I checked and there doesn't seem to be anyone referencing this project directly so it doesn't appear to have broken anything. However I did notice there is duplicated source code. Did you diff it to make sure they are in-sync and we didn't miss any fixes?